vms/platformvm/block/executor/manager.go

```diff
@@ -122,6 +123,21 @@ func (m *manager) VerifyTx(tx *txs.Tx) error {
 		return ErrChainNotSynced
 	}
 
+	recommendedPChainHeight, err := m.ctx.ValidatorState.GetMinimumHeight(context.TODO())
+	if err != nil {
+		return err
+	}
+	err = executor.VerifyWarpMessages(
+		context.TODO(),
+		m.ctx.NetworkID,
+		m.ctx.ValidatorState,
+		recommendedPChainHeight,
```

Comment:
For my understanding VerifyTx verifies txs before inclusion in the mempool?

Reply:
correct